build.sbt

```diff
@@ -29,7 +29,7 @@ lazy val commonSettings = Seq(
   coverageEnabled in Test := true,
   coverageMinimum := 60,
   coverageFailOnMinimum := true,
-  coverageExcludedPackages := "cromwell.pipeline.database.*"
+  coverageExcludedPackages := "<empty>; cromwell\\.pipeline; cromwell\\.pipeline\\.database.*; "
```

Comment:
Could you please describe what it does?

Reply:
Why there is `empty` in front of the string?

Reply:
This are regex to exclude some files from coverage:
I used escape slashes (\\) to make dots match literally.
<empty>  uses for pick up first exclusion. To be honest, it works perfect without it too, but I think there are cases when you really need this tag. And it's in examples in scoverage docs but without any description.
cromwell\\.pipeline excludes files straight from cromwell.pipeline, but not from cromwell.pipeline.database, for example
and cromwell\\.pipeline\\.database.* excludes everything from cromwell.pipeline.database and inside packages (but it is from some other commit, I just put escape slashes there)